custom-middleware/app-level/server.js

```diff
@@ -0,0 +1,69 @@
+import express from "express";
+import { StatusCodes } from "http-status-codes";
+
+const app = express();
+
+const authenticateUser = (req, res, next) => {
```

Comment:
nitpick: it's great to see that these are in their own functions, could we increase the clarity and reduce the cost of reading the code by putting them in their own files?